fdbserver/BlobManager.actor.cpp

```diff
@@ -5145,6 +5134,22 @@ ACTOR Future<Void> bgConsistencyCheck(Reference<BlobManagerData> bmData) {
 	}
 }
 
+ACTOR Future<Void> backupManifest(Reference<BlobManagerData> bmData) {
+	if (g_network->isSimulated() && g_simulator->speedUpSimulation) {
+		return Void();
+	}
+
+	bmData->initBStore();
+	loop {
+		bool pendingSplit = wait(hasPendingSplit(bmData));
+		if (!pendingSplit) {
```

Comment:
Just reiterating this, a large cluster with write traffic will almost always have an active pending split, so this is something we'll need to handle down the road.

Reply:
got it. it's on the plan and it's not an easy issue to address. If we backup manifest while we have pending split, we'll need to address inconsistencies during restore. I pushed the problem to later of this iteration.